### ISSUE TITLE
Would you like to provide the max_acc.pth?

### ISSUE THREAD

**Issue by h1deOnBush** [duanyiqun/DS-DDPM#1]
Hi,
I'd be appreciate if you can provide the max_acc.pth
https://github.com/DuanYiqun/DS-DDPM/blob/633ce40323ecb15e85edb293a4903bbaa3fd0228/comon_two_stream_sep_gaussion_organal_arc_margin_nomalizedy.py#L127

**Comment by duanyiqun** [comment]
Yeah, for sure, I can provide the pre-trained max_acc.pth and attach the link below.
But as we all know that EEG signals quite rely on dataset distribution. I just generate the training data from the BCI-IV 2a data set to let the model distinguish the subject label. If you using your own data, I strongly recommend training a new model on your own data.

**Comment by h1deOnBush** [comment]
ok，no problem， I want to run this code on BCI-IV 2a，many thanks!

**Comment by duanyiqun** [comment]
I uploaded the weights here  https://github.com/DuanYiqun/DS-DDPM/blob/main/assets/max_acc.pth

**Comment by h1deOnBush** [comment]
thanks for providing the max_acc.pth，now I can run the training entry scripts `comon_two_stream_sep_gaussion_organal_arc_margin_nomalizedy.py` successfully。buf I don't know how to generated 
synthetic EEG signals with separated noise. Would you give me some guidance ？

**Comment by duanyiqun** [comment]
The generating process is the same as using image DDPMs, 
If you just want to sample the subject difference from noise. Please see 
https://github.com/DuanYiqun/DS-DDPM/blob/397d19098c6fd15a2c2a9ba5988fe14db89929d1/sample_save_bci_with_subject_noise.py#L592

I rush this code in a hurry, yet the sampling part is not well organized for direct denoising. I will make an update recently. for that part.

**Comment by h1deOnBush** [comment]
appreciate that and looking forward to it

**Comment by BinaryHuang** [comment]
Hi,
I'd be appreciate if you can provide the "/data/bci_iv//single_sep/single_subject_data_1.mat"，many thanks

**Comment by duanyiqun** [comment]
@h1deOnBush hi, sorry for such a late refactoring. 
I'm quite busy with other projects previously.  Please see this file for sampling signals from the trained netowrk https://github.com/duanyiqun/DS-DDPM/blob/main/sample_save.py

**Comment by h1deOnBush** [comment]
it'ok many thx!
